app/views/users/new.html.haml

```diff
@@ -1,14 +1,43 @@
+- content_for :head do
+  %script(type="text/javascript" src="https://js.stripe.com/v2/")
+  %script(type="text/javascript")
+    Stripe.setPublishableKey("#{ENV['STRIPE_PUBLISHABLE_KEY']}");
+
+= javascript_include_tag 'payments'
```

Comment:
I prefer to have javascript in head. But its not a big deal even if its here. (I have it indented 1 level deeper in `content_for` block).